interface/usergroup/facilities.php

```diff
@@ -92,48 +92,92 @@
         facility_npi='" . trim(formData('facility_npi')) . "',
         attn='" . trim(formData('attn')) . "' ,
         primary_business_entity='" . trim(formData('primary_business_entity')) . "' ,
-        tax_id_type='" . trim(formData('tax_id_type')) . "' 
+        tax_id_type='" . trim(formData('tax_id_type')) . "'
     where id='" . trim(formData('fid')) . "'" );
 }
 
 ?>
 <html>
 <head>
-<?php 
-  call_required_libraries(array("jquery-min-3-1-1","bootstrap","fancybox"));
-    resolveFancyboxCompatibility();
+<?php
+  call_required_libraries(array("jquery-min-3-1-1","bootstrap","font-awesome","jquery-ui","iziModalToast"));
 ?>
 <script type="text/javascript" src="<?php echo $GLOBALS['webroot'] ?>/library/js/common.js"></script>
 
 <script type="text/javascript">
 
 $(document).ready(function(){
 
-    // fancy box
-    enable_modals();
+    //iziModal for adding new facilities
+    $(".addFacilities").on("click", function (event) {
+      event.preventDefault();
+      $("#addFacilities-iframe").iziModal('open');
+    });
 
-    // special size for
-    $(".addfac_modal").fancybox( {
-        'overlayOpacity' : 0.0,
-        'showCloseButton' : true,
-        'frameHeight' : 460,
-        'frameWidth' : 650        
+    $("#addFacilities-iframe").iziModal({
+      title: 'Add Facility',
+      subtitle: 'Enter details about the new facility',
+      headerColor: '#88A0B9',
+      closeOnEscape: true,
+      fullscreen: true,
+      overlayClose: false,
+      closeButton: true,
+      theme: 'light',
```

Comment:
No other way of calling this stuff?  Is the "light" theme an LEHR light theme, or a component theme?

Reply:
It's a theme of iziModal itself.